### DISCUSSION TITLE
Easy way to switch custom servers

### DISCUSSION THREAD

**Discussion by goodluck1982** [rustdesk/rustdesk#5934] [Feature Request]
Previously, with version 1.1.9, I used the customized file name   `rustdesk-host=xxx,key=xxxx.exe` to specify server and key. Although this can be done only under windows, it works anyway.  I installed them on `C:\Program Files\RustDesk`.  However, after I installed 1.2.2 today, it **REMOVED**  my previous files such as  `rustdesk-host=xxx,key=xxxx.exe`. This forces me to configure them again and brings trouble to users.

I suggest and expect to implement a customized server list (or config list) in the client which can be selected and specified for each peer.  

Similar to this https://github.com/rustdesk/rustdesk/discussions/5178

Also see https://github.com/rustdesk/rustdesk/discussions/1873

**Comment by rustdesk** [comment]
[[FEATURE] - Connect even if device are not on same server #6198
](https://github.com/rustdesk/rustdesk/issues/6198)
